.azure-pipelines/docker-sonic-slave-template.yml

```diff
@@ -32,89 +35,58 @@ parameters:
   - sonicbld-armhf
 
 jobs:
-- job: Build_${{ parameters.dist }}_${{ parameters.arch }}
+- job: Build_${{ parameters.dist }}_${{ parameters.march }}${{ parameters.arch }}
   timeoutInMinutes: 360
+  variables:
+  - template: .azure-pipelines/template-variables.yml@buildimage
+  - template: .azure-pipelines/azure-pipelines-repd-build-variables.yml@buildimage
   pool: ${{ parameters.pool }}
   steps:
   - template: cleanup.yml
-  - ${{ if eq(variables['Build.Reason'], 'PullRequest') }}:
-    - template: template-clean-sonic-slave.yml
-  - ${{ else }}:
-    - template: '/.azure-pipelines/template-clean-sonic-slave.yml@buildimage'
+  - template: .azure-pipelines/template-clean-sonic-slave.yml@buildimage
   - checkout: self
     clean: true
     submodules: recursive
+  - task: Docker@2
+    displayName: Login to ACR
+    inputs:
+      command: login
+      containerRegistry: ${{ parameters.registry_conn }}
   - bash: |
       set -ex
+      case ${{ parameters.dist }} in
+        bullseye)
+          args='NOJESSIE=1 NOSTRETCH=1 NOBUSTER=1'
+          ;;
+        buster)
+          args='NOJESSIE=1 NOSTRETCH=1 NOBULLSEYE=1'
+          ;;
+        stretch)
+          args='NOJESSIE=1 NOBUSTER=1 NOBULLSEYE=1'
+          ;;
+        jessie)
+          args='NOSTRETCH=1 NOBUSTER=1 NOBULLSEYE=1'
+          ;;
+      esac
+      image_tag=$(make showtag PLATFORM=generic PLATFORM_ARCH=${{ parameters.arch }} $args | grep sonic-slave | tail -n 1)
```

Comment:
How about to replace the linke 57~71 to one line as below?
image_tag=$(BLDENV=${{ parameters.dist }} PLATFORM=generic make -f Makefile.work showtag | grep sonic-slave | tail -n 1)

Reply:
It makes sense.